src/resources/shipment.js

```diff
@@ -110,6 +110,14 @@ export default api => (
       return this.rpc('rates', undefined, undefined, 'get');
     }
 
+    async getSmartrates() {
```

Comment:
Should we camel case `getSmartRates`?

Reply:
I didn't in the Golang library, I suppose if we are making a decision it should be done now and followed everywhere before we get too deep. I'm not super set glued to either one.